engine/apps/email/tasks.py

```diff
@@ -0,0 +1,96 @@
+from socket import gaierror
+
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from django.core.mail import BadHeaderError, get_connection, send_mail
+from django.utils.html import strip_tags
+
+from apps.alerts.models import AlertGroup
+from apps.base.utils import live_settings
+from apps.email.alert_rendering import build_subject_and_message
+from apps.email.models import EmailMessage
+from apps.user_management.models import User
+from common.custom_celery_tasks import shared_dedicated_queue_retry_task
+
+MAX_RETRIES = 1 if settings.DEBUG else 10
+logger = get_task_logger(__name__)
+
+
+@shared_dedicated_queue_retry_task(autoretry_for=(Exception,), retry_backoff=True, max_retries=MAX_RETRIES)
+def notify_user_async(user_pk, alert_group_pk, notification_policy_pk):
+    from apps.base.models import UserNotificationPolicy, UserNotificationPolicyLogRecord
```

Comment:
please comment why imported here

Reply:
Added in https://github.com/grafana/oncall/pull/621/commits/d2d15213b64b85dbb9c5ed070283752b2fa7f4e5